partiql-parser/src/main/antlr/PartiQL.g4

```diff
@@ -766,8 +768,7 @@ functionCall
 
 // SQL-99 10.4 — <routine name> ::= [ <schema name> <period> ] <qualified identifier>
 functionName
-    : (qualifier+=symbolPrimitive PERIOD)* name=( CHAR_LENGTH | CHARACTER_LENGTH | OCTET_LENGTH | BIT_LENGTH | UPPER | LOWER | SIZE | EXISTS | COUNT | MOD )  # FunctionNameReserved
```

Comment:
(self-review) -- this is the rule we were aiming to remove from the ANTLR grammar in this PR